functional-tests/http/src/test/java/org/mule/test/http/functional/requester/ntlm/HttpRequestExpect100ContinueHeaderNtlmAuthTestCase.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2023 Salesforce, Inc. All rights reserved.
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.test.http.functional.requester.ntlm;
+
+import static org.mule.test.http.functional.fips.DefaultTestConfiguration.isFipsTesting;
+
+import static javax.servlet.http.HttpServletResponse.SC_UNAUTHORIZED;
+
+import static org.junit.Assume.assumeFalse;
+
+import org.mule.runtime.core.api.event.CoreEvent;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import io.qameta.allure.Issue;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runners.Parameterized;
+
+import static org.mule.runtime.http.api.HttpHeaders.Names.AUTHORIZATION;
+import static org.mule.runtime.http.api.HttpHeaders.Names.WWW_AUTHENTICATE;
+
+import org.mule.test.runner.RunnerDelegateTo;
+
+@Issue("W-17282518")
```

Comment:
If we are extending a class and using that to execute the assertions, do we add the allure annotation on top of the class ?

Reply:
If all of the children classes will be related with this issue, putting the annotation in the parent class makes sense. In this case, it seems to be the case, so this annotation is ok